### PR TITLE
Re-write: Fix admin panel tabs

### PR DESCRIPTION
**Pull request by tcaiger**
### Changes:

- Pass in correct base url to Toolbar component

### CODE REVIEW

File: packages/admin-panel/src/App.js [hunk 1]
```diff
@@ -17,7 +17,6 @@ import { labelToId } from './utilities';
 
 export const App = ({ user }) => {
   const headerEl = React.useRef(null);
-  const match = useRouteMatch()
```
Comment (tcaiger) [inline packages/admin-panel/src/App.js:20]: the match is reading the context of where the hook is placed so the top level App component is always matching `/`

File: packages/admin-panel/src/App.js [hunk 2]
```diff
@@ -39,24 +38,32 @@ export const App = ({ user }) => {
...
-                    <tab.component getHeaderEl={getHeaderEl} />
-                  </Route>
-                ))}
-                <Redirect to={route.to} />
-              </Switch>
-            </Route>
+            <Route
+              key={route.to}
+              path={route.to}
+              render={({ match }) => {
```
Comment (tcaiger) [inline packages/admin-panel/src/App.js:44]: this is the correct match as it is in the context that the tabs are rendered.

**Review by acdunham — APPROVED** [review]
Looks good to me! Thanks for doing this. Interesting about this one - I tested this when I made this change to the `TabsToolbar` and it worked, but this makes sense. Does it need updating in LESMIS as well?

**Comment by tcaiger** [comment]
> 

Just noting that the tabs did render ok but when clicking on them they errored as the tab link was not working.

**Pull request merged**